datalab/datalab_session/data_operations/fits_output_handler.py

```diff
@@ -22,7 +22,7 @@ class FITSOutputHandler():
     data (np.array): The data for the image HDU.
   """
     
-  def __init__(self, cache_key: str, data: np.array, comment: str=None) -> None:
+  def __init__(self, cache_key: str, data: np.array, comment: str=None, data_header: fits.Header=None) -> None:
```

Comment:
Allow this function to accept a data (SCI) hdu header to place into the image hdu its saving. We should probably extend this a bit more to accept saving more HDUs - I can see wanted to copy over the CAT HDU for certain operations that are looking at the same area, which is basically all of them, so that we can load the source catalog when you make them big. Like if you take a median and then subtract that out from an image you might do that to make the sources disappear or make a single source pop, and seeing the source catalog overlayed might be nice.